server/methods/deleteMessage.coffee

```diff
@@ -18,7 +17,7 @@ Meteor.methods
 
 		deleteQuery = 
 			_id: message._id
-		deleteQuery['u._id'] = Meteor.userId() if user?.admin isnt true 
+		#deleteQuery['u._id'] = Meteor.userId() if user?.admin isnt true 
```

Comment:
@marceloschmidt now is `if user is admin or moderator` instead `if user is admin`, but the line is still required to prevent normal users to delete messages from another users.